src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Toggles/BitToggleTest.razor

```diff
@@ -0,0 +1,11 @@
+﻿<CascadingValue Value="Visual">
+	<BitToggle IsEnabled="@IsEnabled" IsChecked="@IsChecked" IsInlineLabel="@IsInlineLabel" OnText="@OnText" OffText="@OffText"></BitToggle>
```

Comment:
all 2-way bound parameters should be used with "@bind-" unless there is a certain reason for not doing that.

Reply:
Inside toggle component `IsChecked` only changes and other properties not change.
I realized that only the properties that change inside the component should be 2-way bound.